src/Game.java

```diff
@@ -0,0 +1,91 @@
+import java.io.*;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Game {
+
+    static int currentPlayer = 0;
+    static boolean gameOver = false;
+    static int numPlayers;
+    static Snake snakes;
```

Comment:
why does Snake class contains multiple snakes?

Reply:
**_static Snake snakes;_** : its just the name of the object of snake class that contains all the snakes (as hashmap, i.e., head as the key and tail as the value).

Reply:
Then it should not be Snake but Snakes. Also better to have Snake and maintain this in the Game/Board class